### DISCUSSION TITLE
Privacy Policy Consent

### DISCUSSION THREAD

**Discussion by mdwest** [CiviWiki/OpenCiviWiki#1412] [Ideas]
Under European law, we need to have explicit consent for our privacy policy. We need to add a link to the privacy policy and an "I agree" button during the account creation process.

**Comment by vtostado** [comment]
Is this issue still open? I can give it a shot :D

**Comment by topguns837** [comment]
Is this issue still open ?

**Comment by brylie** [comment]
We don't have a privacy policy for the time being.
